src/internal.c

```diff
@@ -9804,9 +9804,11 @@ static int ProcessCSR(WOLFSSL* ssl, byte* input, word32* inOutIdx,
 
     #ifdef WOLFSSL_SMALL_STACK
         CertStatus* status;
+        OcspEntry* single;
```

Comment:
Where OcspEntry is used to represent an OCSP SingleResponse, I've named the variable single.